src/scripts/exclusion_term_expansion.py

```diff
@@ -377,6 +363,10 @@ def cli_get_parser() -> ArgumentParser:
         help='Optional. Path to a "component signature" file, which contains a list of class URIs from Mondo\'s '
              '"alignment module" for the ontology, an alignment module being defined as the list of classes we care '
              'about (e.g. all diseases).')
+    parser.add_argument(
```

Comment:
@matentzn Added explicit `--outpath` param. I'm in the middle of checking if it works right now. My primary concern here is making sure that this works in the situation where we have a `make` target that has two files in the target, e.g. `$(REPORTDIR)/%_term_exclusions.txt $(REPORTDIR)/%_exclusion_reasons.robot.template.tsv:`.

Reply:
THANK YOU!